src/app/components/SearchBar/SearchBar.tsx

```diff
@@ -20,7 +20,13 @@ const Z_INDEX_OFFSET = 2 // Set a custom offset
 const StyledSearchBarToggle = styled(SearchBarToggle)`
   z-index: 23;
   position: absolute;
-  left: 0;
+
+  ${({ searchIsVisible }: { searchIsVisible: boolean }) =>
```

Comment:
So this could be removed
```suggestion
  ${({ searchIsVisible }) =>
```